plugin/bufsize/README.md

```diff
@@ -0,0 +1,34 @@
+# bufsize
+## Name
+*bufsize* - modifies EDNS0 buffer size for protecting answers from vulnerability of IP flagmentation.
+
+## Description
+*bufsize* can limit requester's UDP payload size.  
+It prevents IP fragmentation so that to be ready for the DNS Flag Day 2020 and to deal with DNS vulnerability.
+
+## Syntax
+```text
+bufsize [SIZE]
+```
+
+**[SIZE]** is an int value for setting the buffer size.  
```

Comment:
shouldn't this take into account ipv6 and ipv4 and things like that? Should we allow a SIZE at all, or just provide the defaults?

Reply:
In the Flag Day page, the recommended value is 1232 bytes, it covers both IPv4 and IPv6.
So I decided to accept one option regardless of address family. And it's simple too.
But adding option to classify by an address family could be useful!
If you think this feature is needed, I will implement it. Or could be a feature request for future?

Reply:
did it, OK. Let's just leave one - we have to make this clear in the docs though

Reply:
Thank you for the review! I'll mention it on **Syntax** section in README. How about something like that?
"Only one argument is acceptable, and it covers both IPv4 and IPv6."